index.js

```diff
@@ -132,6 +132,9 @@ exports = module.exports = function bootLoopBackApp(app, options, callback) {
 /**
  * Compile boot instructions and add them to a browserify bundler.
  * @param {Object|String} options as described in `bootLoopBackApp` above.
+ * @property {String} [appId] Application identifier used to load the correct
```

Comment:
Are we planning to use this id to distinguish between applications at runtime? If not, then I am proposing to use `bundleId` as the option name.